<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddId</class>
 <widget class="QWidget" name="AddId">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
font-style:MingLiU-ExtB;
}

QLineEdit{
border-radius:10px;
}

QPushButton#save{
border-style:outset;
font: 75 16pt &quot;宋体&quot;;
font-weight:bold; 
color:rgb(255, 170, 0);
background-color:rgba(225, 225, 225, 0);
}

QPushButton#save:hover{ 
background-color:rgba(225, 225, 225,200);
}
 
QPushButton#save:pressed{
background-color:rgba(225, 225, 225, 200);
}

QPushButton#cancel{
border-style:outset;
font: 75 16pt &quot;宋体&quot;;
font-weight:bold; 
color:rgb(255, 170, 0);
}

QPushButton#cancel{
background-color:rgba(225, 225, 225, 0);
}

QPushButton#cancel:hover{ 
background-color:rgba(225, 225, 225,200);
}
 
QPushButton#cancel:pressed{
background-color:rgba(225, 225, 225, 200);
}
</string>
  </property>
  <widget class="QWidget" name="topwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>2</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>存储</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>321</width>
     <height>151</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="id">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MingLiU-ExtB</family>
     </font>
    </property>
    <property name="text">
     <string>服务器ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="username">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MingLiU-ExtB</family>
     </font>
    </property>
    <property name="text">
     <string>用户名:</string>
    </property>
   </widget>
   <widget class="QLabel" name="password">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MingLiU-ExtB</family>
     </font>
    </property>
    <property name="text">
     <string>密码:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
